src/anonymous.jl

```diff
@@ -12,22 +12,27 @@ else
 const _uncompress = Base._uncompressed_ast
 end
 
+if VERSION < v"1.6-"
 structdata(meth::Method) =
   [meth.module, meth.name, meth.file, meth.line, meth.sig, getfield(meth, syms_fieldname),
    meth.ambig, meth.nargs, meth.isva, meth.nospecialize,
    _uncompress(meth, meth.source)]
+else
+structdata(meth::Method) =
+  [meth.module, meth.name, meth.file, meth.line, meth.sig, getfield(meth, syms_fieldname), 
+   meth.nargs, meth.isva, meth.nospecialize, _uncompress(meth, meth.source)]
+end
 
 initstruct(::Type{Method}) = ccall(:jl_new_method_uninit, Ref{Method}, (Any,), Main)
 
 function newstruct!(meth::Method, mod, name, file, line, sig,
-                    syms, ambig, nargs, isva, nospecialize, ast)
+                    syms, nargs, isva, nospecialize, ast)
```

Comment:
To confirm, has `ambig` been removed?

Reply:
Yeah remove [here](https://github.com/JuliaLang/julia/commit/258d04e8bfdb8189fd61199bccd7641650fec07b#diff-42a3fc6ebd78315e40602ab4108f5cd380ada14ff81987b696d7b4ee9372a4de)